src/internal/abstractButton.js

```diff
@@ -179,7 +179,7 @@ export default {
       }
     },
     getTagName () {
-      const isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') !== -1
+      const isFirefox = typeof navigator !== 'undefined' && navigator.userAgent.toLowerCase().indexOf('firefox') !== -1
```

Comment:
`lodash.toLower` used only on server-side. How you can see, `navigator.userAgent.toLowerCase()` will called only if `navigator` is defined, that works on client-side only. Currently [all browsers have String.prototype.toLowerCase method](http://caniuse.com/#search=tolowercase). Therefore, there is no need to replace

P.S.: I don't understand why Node.js doesn't have an implementation of such a simple method, but it's fact